### ISSUE TITLE
0.1.11 Crashed 2 times

### ISSUE THREAD

**Issue by Od1waN** [SemkoDev/bolero.fun#27]
I updated to 0.1.11 (Windows) and Bolero crashed always. Bolero frontend crashed to a White Screen after the RAM reached approx 2,6 GB. Then the RAM reached approx 5GB and Bolero crashed completely.

![bolero1](https://user-images.githubusercontent.com/21100937/34713526-d9b7e204-f526-11e7-88c3-96107798285a.JPG)

![bolero2](https://user-images.githubusercontent.com/21100937/34713529-dbaad080-f526-11e7-9dad-9f3c51f0e9fc.JPG)

**Comment by romansemko** [comment]
Hmm, could you try 0.1.12 and report if the problem persists?

**Comment by Od1waN** [comment]
Yes the same Problem. Only the frontend crashed when its open. When i Close Bolero it works in Background. I can open Bolero again and the RAM is growing up over 2Gb and i see the White Screen again.

**Comment by sloganking** [comment]
I believe I have the same problem. (Win10)(0.1.12)

This: https://i.imgur.com/c7nNm1d.png

Turns to this: https://i.imgur.com/KLZWcLe.png

After ram usage goes to about 2500MB

After the GUI goes white, ram usage does drop to around 1600MB before slowly climbing over 3500MB and then crashing. Crashing only happens when application GUI is showing. Application does not crash when minimized to system tray.

https://i.imgur.com/sbgOTj8.png

When minimized, application system usage stays like this: 
https://i.imgur.com/VocDWzq.png
https://i.imgur.com/VteGW5Q.png

It's worth noting that the light wallet for me gets "connection refused" and using an online port checker, it shows port 15600 and 16600 are open but says that nothing is running on 14600.

Performance issues could be related to the amount of print out lines stored in the bell notification Icon. As after a while, the application will simply not respond after you click on it, and it seems to loose performance when more lines are printed out.

**Comment by romansemko** [comment]
it seems that it's an IRI issue. It goes into overdrive for some reason.

I have updated IRI to 1.4.1.7 in Bolero 0.1.13. Could you please download and see if you see any difference or crashes?

**Comment by sloganking** [comment]
https://i.imgur.com/4HmIMBW.png
v0.1.13

Constantly getting these error messages as ram goes up. I think this is the primary problem for me when I try running bolero on Windows10. I also think ram is going up because all these error messages are stored in ram (could be wrong).

Gui becomes unresponsive as ram goes up and crashes after ram usage hits 4000MB

https://i.imgur.com/Z7893Qy.png.

Wondering if this is this the version of java I have installed or a problem with bolero.

Edit: 
the crash
https://i.imgur.com/tYxo3QU.png
https://i.imgur.com/Kj3aJik.png

**Comment by romansemko** [comment]
I guess I will have to add logs as a file to be able to see the whole error stack trace. It seems to come from IRI, but I wonder what error it shows exactly at the top...

Will try to add file log for the next release.

**Comment by sloganking** [comment]
This issue on Win10 seems to be fixed for me on version 0.1.14.

It is worth noting that I deleted two folders recoomended by neils12 on the discord #bolero-node channel

"Maybe good to know: i had to delete C:\Users<username>\AppData\Roaming\bolero.fun folder and  C:\Users<username>.bolero folder (of previous tries with older versions), do fresh start and it was fully synced after 10 minutes. "

and there was an iri update around the same time 0.1.14 was released.

Curious if this is fixed for other users as well.
